edifikana/front-end/shared-compose/src/commonMain/kotlin/com/cramsan/edifikana/client/lib/features/root/auth/signup/SignUpScreen.kt

```diff
@@ -79,47 +107,86 @@ internal fun SignInV2Content(
         contentAlignment = Alignment.Center,
     ) {
         Column(
-            modifier = Modifier.padding(Padding.MEDIUM),
+            modifier = Modifier
+                .sizeIn(maxWidth = Size.COLUMN_WIDTH)
+                .padding(Padding.MEDIUM),
             verticalArrangement = Arrangement.spacedBy(Padding.MEDIUM),
             horizontalAlignment = Alignment.CenterHorizontally,
         ) {
-            uistate.signUpForm.errorMessage?.let {
-                Text(it)
+            AnimatedContent(
+                uistate.signUpForm.errorMessage,
+                transitionSpec = {
+                    fadeIn()
+                        .togetherWith(
+                            fadeOut()
+                        )
+                },
+            ) {
```

Comment:
This is for adding an animation for when the error message shows up. Creates a nice fade-in.